build.gradle

```diff
@@ -96,6 +96,7 @@ tasks.withType(Test) {
 }
 
 task integration(type: Test) {
+    jvmArgs = ['-Xms1024m', '-Xmx2048m']
```

Comment:
it works fine for me without jam args above not sure if it will work in the pipeline without those args ?